### DISCUSSION TITLE
Mistral AI released the 8x7B MOE model

### DISCUSSION THREAD

**Discussion by czkoko** [ggml-org/llama.cpp#4379] [General]
https://twitter.com/MistralAI/status/1733150512395038967

[magnet:?xt=urn:btih:5546272da9065eddeb6fcd7ffddeef5b75be79a7&dn=mixtral-8x7b-32kseqlen&tr=udp%3A%2F%http://2Fopentracker.i2p.rocks%3A6969%2Fannounce&tr=http%3A%2F%http://2Ftracker.openbittorrent.com%3A80%2Fannounce](magnet:?xt=urn:btih:5546272da9065eddeb6fcd7ffddeef5b75be79a7&dn=mixtral-8x7b-32kseqlen&tr=udp%3A%2F%http://2Fopentracker.i2p.rocks%3A6969%2Fannounce&tr=http%3A%2F%http://2Ftracker.openbittorrent.com%3A80%2Fannounce)

**Comment by ciekawy** [comment]
https://github.com/ggerganov/llama.cpp/issues/4381
